### DISCUSSION TITLE
Configure object permissions via API

### DISCUSSION THREAD

**Discussion by dan-tabarca** [netbox-community/netbox#8773] [Q&A] [answered]
I am trying to add a new object permission and assign a group for it using curl or requests library from Python. With both options I am receiving the same error: 
`{"groups":["Incorrect type. Expected pk value, received dict."]}`
The syntax that I am using is: 
`curl -s -X POST "https://netbox/api/users/permissions/" -H "Authorization: Token ${token}" --data '{"name": "test","object_types": ["auth.group"]], "groups": [{"id": 1 }], "actions": ["view", "add", "change", "delete"]}'`
It looks like the syntax for the `groups` list is not correct. 
Can you please advise how to correctly POST using this API? I am using Netbox 3.1.5. 
Thank you.

**Comment by jeremystretch** [comment, marked as answer]
I haven't had my coffee yet, but I believe you just need `"groups": [1]`

**Reply by dan-tabarca** [reply]
Yes, that's the way. Thank you!
